### ISSUE TITLE
Feature Request: add image to custom food

### ISSUE THREAD

**Issue by prestidigitonium** [davidhealey/waistline#316]
Hello, when I manually add a new food, it could be useful to add an image (just locally).

**Comment by davidhealey** [comment]
I don't think this is a good idea, it would require using local storage for images and could quickly create issues, especially on devices that don't have a lot of storage space.

**Comment by EmilJunker** [comment]
Maybe the image could be stored as a blob directly in the local database in the `image_url` field? Although I believe there is also a limit to how much can be stored in there...

**Comment by davidhealey** [comment]
@EmilJunker It would still require additional storage space.

**Comment by PhoenixIV** [comment]
I have a different view on this @davidhealey :

If the user wants images he is self-responsible to be aware of the storage space. I would go as far as enabling local storage of web images, as I love being offline.

More than that, how much does a thumbnail need?
